frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewerInputContainer/viewerInputContainer.component.tsx

```diff
@@ -25,13 +25,13 @@ import { VIEWER_ROUTE, ViewerParams } from '@/v5/ui/routes/routes.constants';
 import { TicketsCardHooksSelectors } from '@/v5/services/selectorsHooks';
 import { Overlay, Link } from './viewerInputContainer.styles';
 
-export const ViewerInputContainer = (props) => {
+export const ViewerInputContainer = ({ inputRef = undefined, ...props }) => {
 	const { isViewer, containerOrFederation } = useContext(TicketContext);
 	const { teamspace, project } = useParams<ViewerParams>();
 	
 	const ticketId = TicketsCardHooksSelectors.selectSelectedTicketId();
 
-	if (isViewer) return (<InputContainer {...props} />);
+	if (isViewer) return (<InputContainer ref={inputRef} {...props} />);
```

Comment:
what happened here? there was something not working?

Reply:
Some components were complaining about the `ref` passed to the child input not being populated